### ISSUE TITLE
"directional horizontal swipe" to pass 2.5.7: Dragging Movements?

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2406]
Fourth example in G219 talks about using a "directional horizontal swipe" to pass 2.5.7: Dragging Movements

This doesn't seem Sufficient as a technique for the User and Functional Needs this SC is trying to tackle...

https://w3c.github.io/wcag/techniques/general/G219

> In a Kanban implementation for process management, tasks can be dragged horizontally from one ‘swimming lane’ to another in order to change the status of tasks. **A directional horizontal swipe over an item moves it to the adjacent lane**. This process can be repeated until the item is in the desired lane.

**Comment by jake-abma** [comment]
O chips, in single-pointer we have "path-based gestures" resulting in this one going 'wrong' again... so we say sufficient to use a path-based gesture for Dragging Movements, BUT FAILING 2.5.1 Pointer Gestures AT THE SAME TIME

haha, :-)

**Comment by detlevhfischer** [comment]
@jake-abma 
> sufficient to use a path-based gesture for Dragging Movements, BUT FAILING 2.5.1 Pointer Gestures AT THE SAME TIME

keep laughing... that is the situation all the time with all sorts of SCs...

2.5.1 and 2.5.7 may be combined in the future as the requiements are essentially the same - and the somewhat UA/OS dependent fiddly distinction between pointer gesture and dragging movement can then fall.

**Comment by jake-abma** [comment]
@detlevhfischer we just should delete that (sufficient) example, not propose an example which fails a "single A" SC

**Comment by detlevhfischer** [comment]
@jake-abma But the point it that it may **not** necessarily fail it if some altenative is present to satidfy 2.5.1 (which may be a redundant way of implementing the function executed via the swipe dragging alternative, e.g., the choice form a pop up menu).

**Comment by jake-abma** [comment]
@detlevhfischer you mentioned:

> may not necessarily fail 2.5.1 if some alternative single pointer option is also present 

So an idea to 'add / extend this to the example'?

> A directional horizontal swipe over an item moves it to the adjacent lane and an alternative single pointer option is also present

**Comment by alastc** [comment]
Hi @jake-abma, that example has been updated, so I'll close this one. Please reopen if I've misunderstood.
